packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -939,6 +948,9 @@ class PhysicalModelLayer extends ContainerLayer {
   /// (as described at [Layer]).
   Path clipPath;
 
+  /// Whether and how to clip its children.
```

Comment:
"its" doesn't make sense here since there's no antecedent.

(I'd probably just use the macro.)

Reply:
Done.